From 3c5c1c704ce240b2b998f7d5fcf98d61299cf8f6 Mon Sep 17 00:00:00 2001
From: Pierre-Yves <pyu@riseup.net>
Date: Wed, 17 Jun 2020 18:05:28 +0200
Subject: [PATCH] Add missing include

Signed-off-by: Pierre-Yves <pyu@riseup.net>
---
 src/httpserver.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/httpserver.cpp b/src/httpserver.cpp
index d36c74b..2d83538 100644
--- a/src/httpserver.cpp
+++ b/src/httpserver.cpp
@@ -13,6 +13,7 @@
 #include <sync.h>
 #include <ui_interface.h>
 
+#include <deque>
 #include <memory>
 #include <stdio.h>
 #include <stdlib.h>
-- 
2.27.0

